.gitignore

```diff
@@ -1,5 +1,8 @@
 /node_modules/
 /out/
```

Comment:
general question: were you able to run this in esm mode?

Reply:
yes